src/modules/dashboard/components/ColonyHome/ColonyMeta/ColonyInvite.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import ClipboardCopy from '~core/ClipboardCopy';
+
+import styles from './ColonyInvite.css';
+
+const MSG = defineMessages({
+  inviteLabel: {
+    id: 'dashboard.ColonyHome.ColonyMeta.inviteLabel',
+    defaultMessage: 'Invite your community to join',
+  },
+  copyLabel: {
+    id: 'ClipboardCopy.copyLabel',
+    defaultMessage: `{valueIsCopied, select,
+      true {copied}
+      false {copy}
+    }`,
+  },
+});
+
+interface Props {
+  colonyName: string;
+}
+
+const displayName = 'dashboard.ColonyHome.ColonyInvite';
```

Comment:
Shouldn't this be `dashboard.ColonyHome.ColonyMeta. ColonyInvite`? The i18n messages could use the same treatment (I see your note up there about the `id`)